crates/bevy_gizmos/src/arcs.rs

```diff
@@ -47,15 +47,15 @@ impl<'w, 's, T: GizmoConfigGroup> Gizmos<'w, 's, T> {
         direction_angle: f32,
         arc_angle: f32,
         radius: f32,
-        color: LegacyColor,
+        color: impl Into<Color>,
     ) -> Arc2dBuilder<'_, 'w, 's, T> {
         Arc2dBuilder {
             gizmos: self,
             position,
             direction_angle,
             arc_angle,
             radius,
-            color,
+            color: color.into(),
```

Comment:
Might need to do some performance testing to confirm, but we probably need to go back over the various `From` implementations an annotate them with `#[inline]`. I'm not sure how good the Rust compiler is at inlining these tiny functions.

Reply:
Yeah agreed. I want to do perf testing towards the end of this process.